gnovm/tests/files/assign_unnamed_type/more/realm_compositelit_filetest.gno

```diff
@@ -155,6 +155,25 @@ func main() {
 //         },
 //         {
 //             "T": {
+//                 "@type": "/gno.PointerType",
+//                 "Elt": {
+//                     "@type": "/gno.RefType",
+//                     "ID": "gno.land/r/test.Int"
+//                 }
+//             },
+//             "V": {
+//                 "@type": "/gno.PointerValue",
+//                 "Base": {
+//                     "@type": "/gno.RefValue",
+//                     "Hash": "3c89d875f7d6daa94113aa4c7e03432ba56202c2",
+//                     "ObjectID": "a8ada09dee16d791fd406d629fe29bb0ed084a30:4"
+//                 },
+//                 "Index": "0",
+//                 "TV": null
+//             }
+//         },
+//         {
+//             "T": {
```

Comment:
the order of var declarations has changed a bit; 
the order now reflects the declaration order on file, rather than by dependency.
initStaticBlocks vs predefineNow order.